service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseService.java

```diff
@@ -0,0 +1,109 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Maps;
+import org.json.JSONException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.event.EventListener;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.ccd.client.CoreCaseDataApi;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDataContent;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.client.model.Event;
+import uk.gov.hmcts.reform.ccd.client.model.StartEventResponse;
+import uk.gov.hmcts.reform.document.domain.Document;
+import uk.gov.hmcts.reform.fpl.events.SubmittedCaseEvent;
+
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * handle case submission to take in submitted event and add pdf to CCD.
+ */
+@Component
+public class CaseService {
```

Comment:
Let's rename to `SubmittedCaseEventHandler`. We will have one handler per event. 

Let's also move it into `handlers` package.